### DISCUSSION TITLE
哨站的罗兰大叔是不是有什么不可告人的往事

### DISCUSSION THREAD

**Discussion by kesiwumola** [share121/inter-knot#2247] [其他]
![20240803193945](https://github.com/user-attachments/assets/5d4f2da0-139c-4285-b1ed-9c1adf610615)
![R-C](https://github.com/user-attachments/assets/58eb9ae3-9b23-4fc5-b739-d25ec8077f53)
![20240803193945](https://github.com/user-attachments/assets/a7845238-6cbf-4797-87d8-4ea6360590cf)

**Comment by icenow188** [comment]
不是哥们？<img src="https://img-static.mihoyo.com/communityweb/upload/4a102d21856350263e83278ac8c48526.png" alt="吃惊" width="75" >
